docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -213,22 +218,22 @@ defs = Definitions(
 )
 ```
 
-Using source assets has a few advantages over having the code inside of an asset's op load the data:
+Using source assets has a few advantages over having the code inside of an asset's function load the data without specifying any dependencies:
```

Comment:
[1]

reworked this sentence in general since "having the code inside an assets' op load the data" kinda undermines the `deps` workflow where the asset's op might do just that. I leaned more into the specifying dependencies vs not specifying, but i'm not sure i totally got the idea across quite right. TODO - return to this sentence and re-work